arm-network/2016-09-01/swagger/network.json

```diff
@@ -6469,6 +6612,79 @@
         }
       }
     },
+    "BgpPeerStatus": {
+      "properties": {
+        "localAddress": {
+          "type": "string",
+          "description": "Gets or sets the virtual network gateway's local address"
```

Comment:
>Gets or sets [](start = 26, length = 12)

Please remove "Gets or sets" from descriptions - the language generator will make appropriate text regarding property accessors. (applies to all other descriptions)